roles/kube-burner/files/metrics-aggregated.yaml

```diff
@@ -182,3 +182,6 @@ metrics:
   
   - query: go_gc_duration_seconds{job=~"apiserver|api|etcd",quantile="1"}
     metricName: goGCDurationSeconds
+
+  - query: topk(10,sum(ALERTS{severity!="none"}) by (alertname, severity))
```

Comment:
Maybe we can remove sum from the query to have all labels?
Usint the original query:
![image](https://user-images.githubusercontent.com/4614641/122838531-8760fc00-d2f6-11eb-9ec0-1499b0882c70.png)

Without sum
![image](https://user-images.githubusercontent.com/4614641/122838607-b24b5000-d2f6-11eb-9ffa-b8fc5bd4d088.png)